wire-ios-data-model/Tests/Source/ManagedObjectContext/DatabaseMigrationTests+ParticipantRole.swift

```diff
@@ -19,9 +19,7 @@
 import XCTest
 @testable import WireDataModel
 
-final class DatabaseMigrationTests_Conversations: XCTestCase {
-
-    private let helper = DatabaseMigrationHelper()
+final class DatabaseMigrationTests_Conversations: DatabaseMigrationTestBase {
```

Comment:
question:
Actually I took quite some effort to avoid inheritance in unit tests and follow "composition over inheritance". We had a talk about this in the iOS lounge.
Where do you see benefits to revert it and introduce the inheritance again?

Reply:
I reverted the commit and factored out the duplicated code as a function in the database migration helper.

Reply:
thanks!